vizro-core/docs/pages/user-guides/deploy.md

```diff
@@ -0,0 +1,195 @@
+# How to deploy your dashboard to production
+
+Typically when you create a dashboard, there are two distinct stages:
+
+1. Development. This is when you build your app. You make frequent changes to your code and want to see how the dashboard looks after each change. You may or may not want to make the dashboard accessible to other people.
+1. Production. When you complete development of your app, you _deploy_ it to production. The dashboard should be accessible to other people.
+
+This page describes methods to run your dashboard _in production_. If your dashboard is still in development then you should read our [guide to development](run.md).
+
+Vizro is built on top of [Dash](https://dash.plotly.com/), which itself uses [Flask](https://flask.palletsprojects.com/). Most of our guidance on how to run a Vizro app in development or production is very similar to guidance on Dash and Flask.
+
+??? note
+    There are many possible workflows depending on your requirements. The above describes a simple workflow that applies to many people but is not suitable for everyone. For example:
+
+    - If you are the only user of your app then the process is often simpler since you might never want to deploy to production.
+    - If there are multiple people involved with development then you will need some way to coordinate code changes, such as a [GitHub repository](https://github.com/) or [Hugging Face Space](https://huggingface.co/spaces).
+    - You might want to make your app accessible to only some people using _authentication_.
+    - You might want to update your dashboard after it has been put into production. There is then a cycle of repeated development and deployment.
+    - There might be additional stages or _environments_ for Quality Assurance (QA) to test that the app works correctly before it is deployed.
+
+## Overview
+
+When developing your dashboard you typically run it _locally_ (on your own computer) using the Flask development server. When you deploy to production, this is no longer suitable. Instead, you need a solution that can handle multiple users in a stable, secure and efficient way.
+
+Vizro is designed to be a production-ready framework, which means that the dashboard you created during development is immediately suitable for deployment to production with minimal changes. Under the hood, Vizro benefits from [Dash's stateless architecture](https://dash.plotly.com/sharing-data-between-callbacks#dash-is-stateless), which makes it simple to scale your app to thousands of concurrent users.
```

Comment:
```suggestion
Vizro is a production-ready framework, which means that the dashboard created during development is immediately suitable for deployment to production with minimal changes. Under the hood, Vizro uses [Dash's stateless architecture](https://dash.plotly.com/sharing-data-between-callbacks#dash-is-stateless), designed for scaling to thousands of concurrent users.
```

Reply:
Minor changes but I felt this verged on slightly promotional so I made a couple of changes. It's OK as it is, so ignore me if you want!